### ISSUE TITLE
Directory/file specfication in JMP settings file is problematic

### ISSUE THREAD

**Issue by aymanhab** [aymanhab/opensim-gui-rcnl#34]
In GUI-generated JMP settings files, the input and output directories and files are not written correctly and cause JMP to crash

**Comment by aymanhab** [comment]
@RobSalati can you comment if this is the same or similar to the issue we had earlier with another tool where the underlying Matlab code was expecting relative path while the GUI was not aware of that? Thank you

**Comment by RobSalati** [comment]
Yes JMP expects that it's given a relative directory. I have the error message below. It says it's looking for a directory ".\C:\...\model.osim"
![image](https://github.com/aymanhab/opensim-gui-rcnl/assets/142843461/e24ce30a-a6e0-4f71-957e-70ee3cdd3bcf)

**Comment by RobSalati** [comment]
Marking this as closed. This was fixed on our end.
